fe/fe-core/src/main/java/org/apache/doris/nereids/stats/ExpressionEstimation.java

```diff
@@ -56,12 +62,20 @@ public ColumnStat visit(Expression expr, StatsDeriveResult context) {
         return expr.accept(this, context);
     }
 
+    public ColumnStat visitCaseWhen(CaseWhen caseWhen, StatsDeriveResult context) {
+        throw new RuntimeException("ExpressionEstimation case-when not implemented");
+    }
+
+    public ColumnStat visitCast(Cast cast, StatsDeriveResult context) {
+        return cast.child().accept(this, context);
+    }
+
     @Override
     public ColumnStat visitLiteral(Literal literal, StatsDeriveResult context) {
-        if (literal.isStringLiteral()) {
+        if (ColumnStat.MAX_MIN_UNSUPPORTED_TYPE.contains(literal.getDataType().toCatalogDataType())) {
             return ColumnStat.UNKNOWN;
```

Comment:
should we return a default value instead of unknown?

Reply:
'unknown' is the default value